Makefile

```diff
@@ -12,7 +12,7 @@ all:
 
 install: install-node install-python install-hooks
 
-build: build-specification build-coordinator build-validator build-proxies
+build: build-specification build-coordinator build-proxies
```

Comment:
this change was to allow me to commit changes - dont have `mvn` installed. can we make it so we dont need maven installed to commit things? i didnt make changes to the validator so doesnt seem to make much sense to rebuild it.

Reply:
Think this has broken the build - it's called by the pipeline as well

Reply:
Why not install maven? It takes like 5 seconds